views/index.etlua

```diff
@@ -1,19 +1,239 @@
 <div class="content">
-    <img src="/static/images/logo.png" alt="Fengari" class="logo">
-    <h1>Fengari</h1>
-
-    <div class="repl">
-        <code id="fengari-console"></code>
-        <div class="fengari-input-container">
-            <label id="fengari-prompt" for="fengari-input">&gt; </label>
-            <textarea class="lua" id="fengari-input" rows="1"></textarea>
+
+    <div class="section section--header">
```

Comment:
Use an actual `<section>`?

Reply:
Now i’m confused: what’s the lua way ? If conditions in parenthesis or not ?

Reply:
lua way is no parenthesis